modules/combined/test/tests/heat_conduction_xfem/tests

```diff
@@ -1,10 +1,12 @@
 [Tests]
-  [./heat_conduction_xfem]
+  [heat_conduction_xfem]
     type = Exodiff
     input = heat.i
     exodiff = 'heat_out.e'
     map = false
     # XFEM requires --enable-unique-ids in libmesh
     capabilities = 'unique_id'
-  [../]
+    # XFEM does not support recover
```

Comment:
is there an issue for this?

Reply:
https://github.com/idaholab/moose/issues/11567 is probably the closest. Want a dedicated one?

Reply:
Nah I think that one is good. Referencing the issue number in the comment would be nice